Fix #93, fix #94. Add front end section and fix up multiplayer section

Danar435 opened the pull request.

SebastianMygind commented on src/problem-solution/frontend.tex:
```diff
@@ -0,0 +1,139 @@
+% textidote: ignore end
+
+The following section is a showcase of the look and feel of the finished product.
+It'll go over every aspect of the website, from the landing page to the game interface, with explanations on how to
```
Generally never use abbreviations in "serious" literature

SebastianMygind commented on src/problem-solution/frontend.tex:
```diff
@@ -0,0 +1,139 @@
+listing is empty, as seen in Figure~\ref{fig:lobby-empty}.
+
+Anyone can create a new game by clicking on the green ``Create New Match'' button in the top right corner of the listings.
+This will create add a new entry to the lobby, as seen in Figure~\ref{fig:lobby-game}.
```
create or add not both

SebastianMygind reviewed (changes requested):
looks good except two minor issues.
Sorry for being pedantic, but i think it goes a long way for the overall quality of the article.

MadsHeilmann commented on src/problem-analysis/multiplayer.tex:
```diff
@@ -1,46 +1,41 @@
 There are however different types of multiplayer games, and while chess is originally a tabletop game, adapting it to a
 digital platform has an effect on the multiplayer aspect of the game.
+This section will go over the different types of multiplayer games, and how the transition from local to online
+multiplayer has an effect on learning chess.
```
Maybe mention that it has a "positive" effect to be more clear

Danar435 replied:
Refrence my other comment.

MadsHeilmann commented on src/problem-analysis/multiplayer.tex:
```diff
@@ -1,46 +1,41 @@
 multiplayer.
-As previously discussed, chess as a tabletop is categorized as a local multiplayer game, while chess as a digital game
-is categorized as an online multiplayer game.
+While the game itself remains the same, the way players interact with each other changes drastically.
+This can have a negative effect of the experience players have when learning and playing chess.
```
Is is not the opposite we are arguing? This sentence reads as if going from local to online is bad for learning. That may be true, but maybe then include a new sentence below saying something like "However, in the following section the benefits....". Right now it is like a "bad" cliffhanger before the next section which argues the opposite.

Danar435 replied:
You're right. The next section explains how online games have both upsides and downsides, so I think that a neutral stance is more correct.

MadsHeilmann reviewed (changes requested):
I think the pictures included in the report should be in light mode, as that just looks nicer on the paper IMO. 

I also think that the front-end section reads to much like "redegørelse" or like a guide. Maybe i am wrong, but should it not have more analysis of why different design choises were made and how they enable learning and so on?

It is also to be considered to only include pictures, wherein there is no reference to the emulator, as that is not part of the product - only the development environment.

SebastianMygind commented:
IMPORTANT:

I have messed with some commits so Mads got some of his commit deleted, so it would probably be best to wait with adding more until it is fixed. AKA please don't merge before we fix the conflicts/deletions.

Danar435 commented:
I've added arguments behind some of the design choices and switched the images to light mode. I'm not sure if they look better, because the background blends in with the paper, but I can revert back if you're against it.

MadsHeilmann commented:
You can maybe add a border around the images? I think that will look better.
